### DISCUSSION TITLE
Retain Deployment.yaml And Service.yaml on Conversion To Advanced Microservice

### DISCUSSION THREAD

**Discussion by GioAc96** [mia-platform/community#505] [Ideas]
### Which area of this project could be most improved?

Console

### Description

Hello,

When converting a microservice to the advanced configuration mode, the UI shows two empty Monaco editor, instances, for the `service.yaml` and `deployment.yaml` files respectively. However,  I always find myself opening the configurations repo to retrieve the content of these two files from the latest commit and paste it on the editor, before doing any modifications. Wouldn't it be a better user experience if the UI switched to the advanced mode and set the content of these two files as per the latest commit?

Thank you,

Giorgio

**Comment by fredmaggiowski** [comment]
Hi, thanks for the feedback; this is actually a design decision we made in the past to provide a friction towards adopting raw manifest configuration for microservices since it shrinks a lot the Console value proposition for workloads management.

**Reply by GioAc96** [reply]
Ah, gotcha. Makes sense.
